lib/nolibc/scanf.c

```diff
@@ -8,6 +8,7 @@
 #include <stdarg.h>
 
 #include <uk/plat/console.h>
+#include <uk/assert.h>
```

Comment:
Now we removed the assertion we probably don't need this. Other than that it LGTM 👍🏼

Reply:
Right, the include is gone now